api/xds_protocol.rst

```diff
@@ -36,43 +36,122 @@ occurs.
 Streaming gRPC subscriptions
 ----------------------------
 
-Singleton resource type discovery
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Resource Types
+~~~~~~~~~~~~~~
 
-A gRPC 
-:ref:`ApiConfigSource <envoy_api_msg_core.ApiConfigSource>`
-can be specified independently for each xDS API, pointing at an upstream
-cluster corresponding to a management server. This will initiate an
-independent bidirectional gRPC stream for each xDS resource type,
-potentially to distinct management servers. API delivery is eventually
-consistent. See :ref:`Aggregated Discovery Service <xds_protocol_ads>`
-below for situations in which explicit control of sequencing is required.
+Every configuration resource in the xDS API has a type associated with it. The following types are supported:
 
-Type URLs
-^^^^^^^^^
-
-Each xDS API is concerned with resources of a given type. There is a 1:1
-correspondence between an xDS API and a resource type. That is:
-
--  LDS: :ref:`envoy.api.v2.Listener <envoy_api_msg_Listener>`
--  RDS: :ref:`envoy.api.v2.RouteConfiguration <envoy_api_msg_RouteConfiguration>`
--  VHDS: :ref:`envoy.api.v2.Vhds <envoy_api_msg_RouteConfiguration>`
--  CDS: :ref:`envoy.api.v2.Cluster <envoy_api_msg_Cluster>`
--  EDS: :ref:`envoy.api.v2.ClusterLoadAssignment <envoy_api_msg_ClusterLoadAssignment>`
--  SDS: :ref:`envoy.api.v2.Auth.Secret <envoy_api_msg_Auth.Secret>`
--  RTDS: :ref:`envoy.service.discovery.v2.Runtime <envoy_api_msg_service.discovery.v2.Runtime>`
+-  :ref:`envoy.api.v2.Listener <envoy_api_msg_Listener>`
+-  :ref:`envoy.api.v2.RouteConfiguration <envoy_api_msg_RouteConfiguration>`
+-  :ref:`envoy.api.v2.route.VirtualHost <envoy_api_msg_route.VirtualHost>`
+-  :ref:`envoy.api.v2.Cluster <envoy_api_msg_Cluster>`
+-  :ref:`envoy.api.v2.ClusterLoadAssignment <envoy_api_msg_ClusterLoadAssignment>`
+-  :ref:`envoy.api.v2.Auth.Secret <envoy_api_msg_Auth.Secret>`
+-  :ref:`envoy.service.discovery.v2.Runtime <envoy_api_msg_service.discovery.v2.Runtime>`
 
 The concept of `type URLs <https://developers.google.com/protocol-buffers/docs/proto3#any>`_ appears below, and takes the form
-`type.googleapis.com/<resource type>`, e.g.
-`type.googleapis.com/envoy.api.v2.Cluster` for CDS. In various
-requests from Envoy and responses by the management server, the resource
-type URL is stated.
+`type.googleapis.com/<resource type>` -- e.g., `type.googleapis.com/envoy.api.v2.Cluster` for a `Cluster` resource. In various
+requests from Envoy and responses by the management server, the resource type URL is stated.
+
+Variants of the xDS API
```

Comment:
Nit: I'd replace API with "transport protocol" here.

Reply:
I don't think we should say "incremental API" anywhere; we can say "incremental xDS". I think in this case at least it makes sense, as you are focusing on the transport specifics.